### ISSUE TITLE
[Bug]: aria-labels, title(Tooltip), target Attributes are not added for Social Icons in Homepage.js File

### ISSUE THREAD

**Issue by NarendraKoya999** [ssitvit/Games-and-Go#878]
### Give a brief about the bug discovered✍

Description:
- `aria-labels, title(Tooltip), target Attributes are not added for Social Icons in Homepage.js File.`
-  These `three attributes are very Important for accessibility Point of View like screen readers`

![gamesandgo-1](https://github.com/ssitvit/Games-and-Go/assets/84512164/5e202d9c-b6ca-4e1e-b15e-0147d7960bf1)

### What is the expected behavior?

Description:
- `Add above Raised attributes for Social Links in Footer Section`
- `It's Improving the SEO Optimization`

### Give a brief description to get the expect behaviour.

Description:
- The aria-label attribute provides a way to place a descriptive text label on an object, such as a link, when there are no elements visible on the page that describe the object. If descriptive elements are visible on the page, the aria-labelledby attribute should be used instead of aria-label .
- The title attribute gives users an important indication as to where they will be directed to when they click on a link. If the user hovers with the mouse over a link reference, the title text, which was defined in the attribute, will be displayed.
- The target attribute specifies where the linked document will open when the link is clicked. The default is the current window. If target="_blank" , the linked document will open in a new tab or (on older browsers) a new window.

### Are you contributing for a program?

GSSoC2023

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/ssitvit/Games-and-Go/blob/main/Contributing.md) of this project.

**Comment by panwar8279** [comment]
Hi @NarendraKoya999 , I have assigned this task to you. Kindly complete your work within 4 days. Good luck!
